be/src/common/config.h

```diff
@@ -481,6 +481,12 @@ namespace config {
     // config for tablet meta checkpoint
     CONF_Int32(tablet_meta_checkpoint_min_new_rowsets_num, "10");
     CONF_Int32(tablet_meta_checkpoint_min_interval_secs, "600");
+
+    // config for default rowset type
+    // 0: ALPHA_ROWSET
+    // 1: BETA_ROWSET
+    CONF_Int32(default_rowset_type, "0");
+    CONF_Int32(compaction_rowset_type, "0");
```

Comment:
It will be better to use string, and we convert string to enum with code. It will make sense.

Reply:
ok